modules/charon-core/src/test/java/org/wso2/charon3/core/utils/PatchOperationUtilTest.java

```diff
@@ -0,0 +1,400 @@
+        ArrayList<AttributeSchema> subAttributeSchemaList = new ArrayList<>();
+        subAttributeSchemaList.add(subAttributeSchema1);
+        subAttributeSchemaList.add(subAttributeSchema2);
+        ArrayList<AttributeSchema> subAttributeSchemaList1 = new ArrayList<>();
+        subAttributeSchemaList1.add(subAttributeSchema3);
+        ArrayList<AttributeSchema> subAttributeSchemaList2 = new ArrayList<>();
+        subAttributeSchemaList2.add(subAttributeSchema4);
+        AttributeSchema attributeSchema1 =
+                SCIMAttributeSchema.createSCIMAttributeSchema(
+                        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User",
+                        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User", COMPLEX,
+                        false, "", false, false,
+                        READ_WRITE, DEFAULT, NONE, null, null, subAttributeSchemaList);
+        AttributeSchema attributeSchema2 =
+                SCIMAttributeSchema.createSCIMAttributeSchema(
+                        "urn:ietf:params:scim:schemas:core:2.0:User:emails",
+                        "emails", COMPLEX,
+                        true, "", false, false,
+                        READ_WRITE, DEFAULT, NONE, null, null, subAttributeSchemaList1);
+        AttributeSchema attributeSchema3 =
+                SCIMAttributeSchema.createSCIMAttributeSchema(
+                        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:country",
+                        "country", STRING, false, "", false, false,
+                        READ_WRITE, DEFAULT, NONE, null, null, null);
+        AttributeSchema attributeSchema4 =
+                SCIMAttributeSchema.createSCIMAttributeSchema(
+                        "urn:ietf:params:scim:schemas:extension:enterprise:2.0:User:manager",
+                        "manager", COMPLEX, false, "", false, false,
+                        READ_WRITE, DEFAULT, NONE, null, null, subAttributeSchemaList2);
+        SCIMResourceTypeSchema scimResourceTypeSchema =
+                SCIMResourceTypeSchema.createSCIMResourceSchema(schemasList, attributeSchema1, attributeSchema2,
+                        attributeSchema3, attributeSchema4);
+
+        return scimResourceTypeSchema;
+    }
+
+    private User getUser() throws InstantiationException, IllegalAccessException {
+
+        User user = User.class.newInstance();
+        user.setSchema("urn:ietf:params:scim:schemas:core:2.0:User");
+        user.setSchema("urn:ietf:params:scim:schemas:extension:enterprise:2.0:User");
+
+        SimpleAttribute simpleAttribute = new SimpleAttribute("nickName", "shaggy");
+        simpleAttribute.setMutability(SCIMDefinitions.Mutability.READ_WRITE);
+        simpleAttribute.setRequired(false);
+        user.setAttribute(simpleAttribute);
+
+        SimpleAttribute simpleAttribute1 = new SimpleAttribute("id",
+                "229d3f0d-a07b-4052-bf4d-3071ecafed04");
```

Comment:
no need to break this line. I doesn't go beyond 120 charactors.

Reply:
fixed in 6e37f6e580f0e7c973edbc994d55b5a8062d5c14